terratest/src/test/pega/pega-environment-config_test.go

```diff
@@ -1,43 +1,41 @@
 package pega
 
 import (
+	"fmt"
 	"github.com/gruntwork-io/terratest/modules/helm"
 	"github.com/stretchr/testify/require"
 	k8score "k8s.io/api/core/v1"
 	"path/filepath"
 	"testing"
-	"fmt"
 )
 
-
-
-func TestPegaEnvironmentConfig(t *testing.T){
-	var supportedVendors = []string{"k8s","openshift","eks","gke","aks","pks"}
-	var supportedOperations =  []string{"deploy","install-deploy","upgrade-deploy"}
-    var deploymentNames = []string{"pega","myapp-dev"}
+func TestPegaEnvironmentConfig(t *testing.T) {
+	var supportedVendors = []string{"k8s", "openshift", "eks", "gke", "aks", "pks"}
+	var supportedOperations = []string{"deploy", "install-deploy", "upgrade-deploy"}
+	var deploymentNames = []string{"pega", "myapp-dev"}
 
 	helmChartPath, err := filepath.Abs(PegaHelmChartPath)
 	require.NoError(t, err)
 
-	for _,vendor := range supportedVendors{
+	for _, vendor := range supportedVendors {
 
```

Comment:
Can you remove new lines between the for loops?

Reply:
Updated code with review comments.